app/util/data_preparation/jira_prepare_data.py

```diff
@@ -153,17 +156,18 @@ def __check_current_language(jira_api):
                          f'Please change your profile language to "English (United States) [Default]"')
 
 
-def main():
-    print("Started preparing data")
-
-    url = JIRA_SETTINGS.server_url
-    print("Server url: ", url)
+def __check_rte_status(jira_api):
+    app_prop = jira_api.get_applications_properties()
+    rte_status = [i['value'] for i in app_prop if i['id'] == 'jira.rte.enabled'][0]
+    return rte_status
```

Comment:
It would be useful to return boolean instead of string. `return rte_status=='true'`

Reply:
Already done )!